docker/helloworld/dockerize

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Yeah, that's a totally fair point. I had considered trying to provide just one Dockerfile but decided against it. The size tradeoff -- 700MB vs 10MB -- felt too large to me.